libs/sections/src/cookie-statement/child/header.section.ts

```diff
@@ -1,4 +1,14 @@
-export const header = () => `
+import { html } from 'lit';
+import type { WebComponent } from '@domg-wc/common-utilities';
+import { VlFunctionalHeaderComponent } from '@domg-wc/components';
+
+export const cookieStatementHeaderElements = (): WebComponent[] => [VlFunctionalHeaderComponent];
+
+// NOTE: Dit is de zelfde component van
```

Comment:
maak hier gerust een nieuw ticket van in JIRA waarin je dit documenteert

Reply:
gedaan - ticket gecreerd https://www.milieuinfo.be/jira/browse/UIG-2789